### ISSUE TITLE
After upgrading to flutter 3,i can't build my app this error message keep coming up

### ISSUE THREAD

**Issue by Trinity6264** [wilburt/flutter_paystack#117]
e: /media/trinity/E24E79554E792387/folders/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_paystack-1.0.5+1/android/src/main/kotlin/co/paystack/flutterpaystack/FlutterPaystackPlugin.kt: (48, 62): Type mismatch: inferred type is Activity? but Activity was expected
e: /media/trinity/E24E79554E792387/folders/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_paystack-1.0.5+1/android/src/main/kotlin/co/paystack/flutterpaystack/MethodCallHandlerImpl.kt: (19, 37): Type mismatch: inferred type is BinaryMessenger? but BinaryMessenger was expected

**Comment by Milimeter** [comment]
iOS works though

**Comment by isaacadariku** [comment]
Yes got same issue

```
e: /Users/isaacadariku/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_paystack-1.0.5+1/android/src/main/kotlin/co/paystack/flutterpaystack/FlutterPaystackPlugin.kt: (48, 62): Type mismatch: inferred type is Activity? but Activity was expected
e: /Users/isaacadariku/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_paystack-1.0.5+1/android/src/main/kotlin/co/paystack/flutterpaystack/MethodCallHandlerImpl.kt: (19, 37): Type mismatch: inferred type is BinaryMessenger? but BinaryMessenger was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_paystack:compileDebugKotlin'.
> Compilation error. See log for more details
```

**Comment by mostafa2014nase** [comment]
i have same error 
![image](https://user-images.githubusercontent.com/84651428/168790184-72928712-76cc-476d-b3b0-d0a1f553f7d6.png)

**Comment by isaacadariku** [comment]
Any one have the issue, before they fix it. Change your `flutter_paystack:` line to this.

```
  # Temporarily using this until we can resolve the issue with the plugin.
  # Issue is: https://github.com/wilburt/flutter_paystack/issues/117
  flutter_paystack:
    git:
      url: https://github.com/definitelyme/flutter_paystack.git
      ref: a4a33c3dd0a12f46d655a2e63d11e9f20ba82d01
```

**Comment by mostafa2014nase** [comment]
same error in some another packages 

> Any one have the issue, before they fix it. Change your `flutter_paystack:` line to this.
> 
> ```
>   # Temporarily using this until we can resolve the issue with the plugin.
>   # Issue is: https://github.com/wilburt/flutter_paystack/issues/117
>   flutter_paystack:
>     git:
>       url: https://github.com/definitelyme/flutter_paystack.git
>       ref: a4a33c3dd0a12f46d655a2e63d11e9f20ba82d01
> ```

same error in some another packages 
I think it's relation with kotlin file 
thanks in advance 
![image](https://user-images.githubusercontent.com/84651428/168795208-5d342944-ea4f-4294-b9a1-d38108297327.png)

**Comment by isaacadariku** [comment]
> > Any one have the issue, before they fix it. Change your `flutter_paystack:` line to this.
> > 
> >             
> >                 ![Save to Pieces](chrome-extension://igbgibhbfonhmjlechmeefimncpekepm/save_button.svg)
> >             
> >             ```
> >   # Temporarily using this until we can resolve the issue with the plugin.
> >   # Issue is: https://github.com/wilburt/flutter_paystack/issues/117
> >   flutter_paystack:
> >     git:
> >       url: https://github.com/definitelyme/flutter_paystack.git
> >       ref: a4a33c3dd0a12f46d655a2e63d11e9f20ba82d01
> > ```
> 
> same error in some another packages I think it's relation with kotlin file or something like that thanks in advance
> 
> ![image](https://user-images.githubusercontent.com/84651428/168795208-5d342944-ea4f-4294-b9a1-d38108297327.png)

Yes, the new upgrade affect those files, but since you have pass through Paystack issue, you can check the other package if there is a solution over there.

**Comment by paultech4u** [comment]
> Any one have the issue, before they fix it. Change your `flutter_paystack:` line to this.
> 
> ```
>   # Temporarily using this until we can resolve the issue with the plugin.
>   # Issue is: https://github.com/wilburt/flutter_paystack/issues/117
>   flutter_paystack:
>     git:
>       url: https://github.com/definitelyme/flutter_paystack.git
>       ref: a4a33c3dd0a12f46d655a2e63d11e9f20ba82d01
> ```

@isaacadariku This works fine for me, was having the same issues after upgrading to 3.0 but after using this it builds successfully thanks.

**Comment by vijayvaghela72** [comment]
This Solution is awesome for now ..

Thank You so much for this

**Comment by wilburt** [comment]
This should be fixed in 1.0.6. Can you update the package version?
